source/guide_grocy.rst

```diff
@@ -0,0 +1,144 @@
+.. highlight:: console
+
+.. author:: Jonas <https://github.com/jfowl>
+
+.. tag:: lang-php
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/grocy.svg
+      :align: center
+
+####
+Grocy
+####
+
+.. tag_list::
+
+Grocy_ is a web-based self-hosted groceries & household management solution for your home.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`Domains <web-domains>`
+
+License
+=======
+
+Grocy is released under the `MIT license`_.
+
+Prerequisites
+=============
+
+Your URL needs to be setup:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+Step 1
+-----
+
+Download grocy to your home directory.
+
+::
+
+ [isabell@stardust isabell]$ wget https://releases.grocy.info/latest -O grocy_latest.zip
+ […]
+ Saving to: ‘grocy_latest.zip’
+ 
+ grocy_latest.zip        100%[===============================>]  54,83M  12,6MB/s    in 4,4s
+
+ 2020-08-26 12:09:31 (12,5 MB/s) - ‘grocy_latest.zip’ saved [57489683/57489683]
+ [isabell@stardust isabell]$ 
+
+Step 2
+------
+
+Unpack the downloaded grocy release
+
+::
+
+ [isabell@stardust isabell]$ unzip -q grocy_latest.zip -d grocy
+ [isabell@stardust isabell]$ rm grocy_latest.zip
+ [isabell@stardust isabell]$ 
+
+
+Step 3
+------
+
+If you want to only host grocy on your Uberspace (best practice since Uberspace 7) you should link your
+DocumentRoot to grocy's public folder:
+
+::
+
+ [isabell@stardust isabell]$ cd /var/www/virtual/$USER/
+ [isabell@stardust isabell]$ rm -r html
+ [isabell@stardust isabell]$ ln -s /home/$USER/grocy/public /var/www/virtual/$USER/html
+ [isabell@stardust isabell]$ 
+
+If you instead want to host grocy alongside other applications, just leave the html folder and create the link inside of it, like this:
+
+::
+
+ [isabell@stardust isabell]$ ln -s /home/$USER/grocy/public /var/www/virtual/$USER/html/grocy
+ [isabell@stardust isabell]$ 
+ 
+Please note that this will require further tweaking in the config file.
+
+Step 4
+------
+
+Copy the default config file to ``data/config.php`` and edit it to your liking
+
+::
+
+ [isabell@stardust isabell]$ cp config-dist.php data/config.php
```

Comment:
Both the shell prompt and the last `cd /var/www/virtual/$USER/` drop us in `/var/www/virtual/$USER/`, but there is no `config-dist.php` in that directory. Is this missing a `cd /home/$USER/grocy`? Please adjust the shell prompt to match as well.